ansi/halfblocks/encoder.go

```diff
@@ -0,0 +1,626 @@
+package halfblocks
+
+import (
+	"fmt"
+	"image"
+	"image/color"
+	_ "image/gif"
+	_ "image/jpeg"
+	_ "image/png"
```

Comment:
We shouldn't import image encodings in libraries, this should happen in the `main` package, and it is the responsibility of the user

Reply:
you are right, I forgot to remove it thx!